layouts/partials/bib/entry/extras.html

```diff
@@ -14,6 +14,9 @@
 {{- with .entry.doi -}}
   <span class="{{ $class }} pubref"><a href="https://doi.org/{{ . }}" target="_blank" rel="noopener">Release</a></span>
 {{- end -}}
+{{- with .entry.code -}}
+  <span class="{{ $class }} pubref"><a href="{{ . }}" target="_blank" rel="noopener">Code</a></span>
```

Comment:
Change the `pubref` class to, say, `coderef`:

```suggestion
  <span class="{{ $class }} coderef"><a href="{{ . }}" target="_blank" rel="noopener">Code</a></span>
```

The classes of different extras’ `<span>`s (beyond the common
`$class`) are supposed to be different to allow for their potential
differentiation in styling.